MathLib/Integration/GaussLegendre.h

```diff
@@ -29,16 +29,27 @@ struct GaussLegendre {
     static const double W[Order];
 };
 
-extern template <> double const GaussLegendre<1>::X[1];
```

Comment:
Nope
```
error: explicit specialization of 'X' after instantiation template <> double const GaussLegendre<1>::X[1] = {0.};
```

I'll set correct `#ifndef _MSC_VER`, and then it's OK. On the other hand one could move all definitions to the headers...